ipatests/test_webui/test_translation.py

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+Test translations
+"""
+
+from ipatests.test_webui.ui_driver import UI_driver
+from ipatests.test_webui.ui_driver import screenshot
+
+try:
+    from selenium.webdriver.common.by import By
+    from selenium.webdriver.support.wait import WebDriverWait
+    NO_SELENIUM = False
+except ImportError:
+    NO_SELENIUM = True
```

Comment:
No reason to write this try... except statement in each test file.

Reply:
As it was found out 'try/except' is used to satisfy pylint.
But "NO_SELENIUM" is no used variable.
Fixed.